docs/en/03_How_tos/use_alternate_module_version.md

```diff
@@ -1,28 +1,42 @@
-title: Using module forks
-summary: How to use an alternate module version (fork)
+title: Use an alternative module version
+summary: How to use an alternate module version
 
-# How to use an alternate module version (fork)
+# How to use an alternate module version
 
-There are situations when you'd like to override the module version provided by the recipe's `composer.json`. One good
-example is if you contribute to the upstream repository, and you would like to use the fixed version straight away.
+There may be a scenario where you want to update to a new version of a commercially supported module outside of the regular CWP quarterly release cadence. Examples may include upgrading to a new module version to take advantage of a security patch that is not yet available in CWP, or you have contributed to the upstream repository and would like access to a specific bug fix or new feature.
 
 To do this we can use a composer's *inline alias*. This is a way of specifying that our desired version is satisfying
 some other requirement (in this case coming from the recipe).
 
-For example, if we wanted to use the `dev-master` version of the `silverstripe-advancedworkflow` module, we would add the
-following statement into the `require` section of our project `composer.json`:
+For example, assuming you want to take advantage of a security patch in a new version of the SilverStripe framework module and your site is currently running a CWP recipe version of `~2.2.0@stable`, your `composer.json` should look like this:
 
 ```
-"symbiote/silverstripe-advancedworkflow": "dev-master as 4.0.1"
+...
+"require": {
+        "cwp/cwp-installer": "2.2.0@stable"
+},
+...
 ```
 
-This will satisfy the `"silverstripe/advancedworkflow": "4.0.1"` requirement embedded in the [SilverStripe collaboration recipe](https://github.com/silverstripe/recipe-collaboration).
+*If you're not using the `cwp/cwp-installer` recipe your `composer.json` may instead reference the `cwp/cwp-core` recipe_*
 
-Please note that generally you should only do this if you are sure about the impact of switching the module version, and
-after you have tested it yourself for regressions with other modules.
+To update to a new version of SilverStripe core, say version `4.3.1`, you should add an alias to the `require` section of the `composer.json` file, like:
 
-Remove this statement as soon as the patch is included in an official release. We suggest to paste the link to the
-revision containing the required change into your commit message so it's easy to find out if the customisation can be
+```
+"silverstripe/framework": "4.3.1 as 4.3.0"
+```
```

Comment:
It's worth mentioning as well that you will then have to run `composer update silverstripe/framework` for Composer to pull in this change and update your `composer.lock` file with the new installation version

Reply:
I've added something in, can you please have another look?